src/OmniSharp.Abstractions/Services/IAssemblyLoader.cs

```diff
@@ -7,6 +7,8 @@ namespace OmniSharp.Services
     public interface IAssemblyLoader
     {
         Assembly Load(AssemblyName name);
+
+        IEnumerable<Assembly> LoadAll(string folderPath);
```

Comment:
I'd prefer this be called `LoadAllFrom` to call out that it uses the LoadFrom context rather than the Load context.